docs/aws/audit/elbmonitoring/rules/elb_web_tier_health_check_remediation.mdx

```diff
@@ -1,6 +1,201 @@
+   python configure_health_check.py
+   ```
+
+This script ensures that the health check configurations for your web-tier ELBs are set correctly, preventing misconfigurations. Adjust the `health_check_config` dictionary values as needed to match your specific requirements.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Log in to the AWS Management Console and navigate to the EC2 dashboard.
+2. In the navigation pane, under LOAD BALANCING, click on Load Balancers.
+3. Select the Elastic Load Balancer (ELB) that you want to check.
+4. In the bottom panel, click on the Health Check tab. Here, you can see the health check configurations for the selected ELB. Check if the configurations are set correctly according to your application's requirements.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. **Install and Configure AWS CLI**: Before you can start, make sure you have AWS CLI installed and configured on your system. You can install it using pip (Python package installer). The command to install AWS CLI is `pip install awscli`. After installing, you can configure it using `aws configure` command. You will be asked to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. **List all the Load Balancers**: Use the following AWS CLI command to list all the load balancers in your account.
+
+    ```
+    aws elbv2 describe-load-balancers
+    ```
+
+    This command will return a JSON output with details of all the load balancers.
+
+3. **Describe Load Balancer Attributes**: For each load balancer, you need to check its attributes. You can do this using the following command:
+
+    ```
+    aws elbv2 describe-load-balancer-attributes --load-balancer-arn <value>
+    ```
+
+    Replace `<value>` with the ARN of your load balancer. This command will return a JSON output with the attributes of the specified load balancer.
+
+4. **Check the Health Check Configuration**: In the JSON output, look for the `HealthCheck` attribute. This attribute contains the health check configuration for the load balancer. Make sure that the `HealthCheckPath` is set to the path where your application's health check page is located, `HealthCheckIntervalSeconds` is set to the time interval for health checks, and `HealthCheckTimeoutSeconds` is set to the amount of time to wait when receiving a response from the health check.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) in your environment. Boto3 allows you to directly create, update, and delete AWS services from your Python scripts.
+
+```python
+pip install boto3
+```
+
+2. Import the necessary libraries and establish a session with AWS using your access and secret keys.
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>